roles/etcd_ca/tasks/main.yml

```diff
@@ -38,9 +38,9 @@
   delegate_to: "{{ etcd_ca_host }}"
   run_once: true
 
-- command: touch {{ etcd_ca_db }}
-  args:
-    creates: "{{ etcd_ca_db }}"
+- file:
+    path: "{{ etcd_ca_db }}"
+    state: touch
```

Comment:
This change will cause `{{ etcd_ca_db }}` to be touched every run while the existing file task only touched (created) the file when it didn't exist.